sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ResolverClient.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Iot.ModelsRepository
+{
+    /// <summary>
+    /// The <c>ResolverClient</c> class supports DTDL model resolution providing functionality to
```

Comment:
Not the right tag to use. Should be <see>, but in this case it is the current type, so it is unecessary.